tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
+    "rootDir": "./",
+    "target": "es5",
     "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
I would prefer to keep number of configs minimal, especially for ones that we don't really use so that it's easier to maintain.